packages/react-docs/UPGRADE-GUIDE.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Upgrade guide
+section: overview
+---
+
+Hey Fliers, we've been busy for the past 12 weeks keeping up with changes to PatternFly's HTML and CSS. We've replaced `babel` with `tsc` as our compiler of choice, removed [prop-types](https://www.npmjs.com/package/prop-types) in favor of using our `.d.ts` types which are supported in most editors. This upgrade guide details **what** was broken and **how** to fix it. We've included links to the pull requests in case you want to follow a trail of **why** a change was made.
```

Comment:
I wish I could always write like you suggest. Thanks. I've updated this!

Reply:
I thought this opening paragraph was really well done! It captures the PatternFly voice perfectly :) I only made a few small tweaks, but otherwise I think you did an awesome job.